style/reading/links-to-physics.less

```diff
@@ -19,15 +20,11 @@
     text-transform: uppercase;
     letter-spacing: 2px;
     text-align: center;
-    background: @physics-background-secondary;
+    background: (@physics-background-secondary);
 
-    &:before {
-      content: "\2014  ";
-    }
+    &:before { content: "\2014  ";}
```

Comment:
1. done
2. yes
3. Very close, probably can be combines, but was waiting for styles for each to be finished, then refactoring as needed. I supposed I could have labeled this PR as WIP-ish.